lib/components/config-validator.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
-var fs = require("fs");
-var yaml = require("js-yaml");
-var JaySchema = require("jayschema");
-var extend = require("extend");
+const fs = require("fs");
+const yaml = require("js-yaml");
+const validator = require('is-my-json-valid');
```

Comment:
quotes are different

Reply:
also can we `UpperCamelCase` the validator given it's a class?

Reply:
but it's a class that gives a function... gah why is js so horrible sometimes :/ I still think upper camel case is the way to go here, even if the package is weird.

Reply:
I was making it conform to the others, but yea